### PR TITLE
Adjust device mode integration for iOS v2

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Adjust iOS v2 device mode integration steps

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

## Notion ticket link

> [Adjust iOS v2](https://www.notion.so/rudderstacks/Documentation-Update-Adjust-Device-Mode-37ec1bd61b86482c920e0a5f96aa1322)

### CODE REVIEW

File: docs/destinations/attribution/adjust.mdx [hunk 1]
```diff
@@ -83,6 +84,62 @@ RudderConfigBuilder *builder = [[RudderConfigBuilder alloc] init];
...
+</ol>
+      </TabPanel>
+      <TabPanel>
+<div class="warningBlock">
+
+This device mode integration is supported for Adjust v4.29.7 and above.
+</div>
+        To add Adjust to your iOS project, follow these steps:
+<ol>
+<li>Install <code class="inline-code">RudderAdjust</code> (available through <a href="https://cocoapods.org/">CocoaPods</a> by adding the following line to your <code class="inline-code">Podfile</code>:
```
Comment (ishachopra0401) [inline docs/destinations/attribution/adjust.mdx:96]:
```suggestion
<li>Install <code class="inline-code">RudderAdjust</code> (available through <a href="https://cocoapods.org/">CocoaPods</a>) by adding the following line to your <code class="inline-code">Podfile</code>:
```

**Review by ishachopra0401 — APPROVED** [review]

File: docs/destinations/attribution/adjust.mdx [hunk 2]
```diff
@@ -83,6 +84,62 @@ RudderConfigBuilder *builder = [[RudderConfigBuilder alloc] init];
...
+</span>
+
+<span>
+
+```objectivec
+@import RudderAdjust;
+```
+</span>
+</li>
+<li>Add the imports to your <code class="inline-code">AppDelegate</code> file under <code class="inline-code">didFinishLaunchingWithOptions</code> method, as shown:
```
Comment (pallabmaiti) [inline docs/destinations/attribution/adjust.mdx:120]: Shouldn't it be 'add the following code to your.......'?

**Review by pallabmaiti — CHANGES_REQUESTED** [review]

File: docs/destinations/attribution/adjust.mdx (hunk 2 above)
Comment (ameypv-rudder) [inline docs/destinations/attribution/adjust.mdx:120]:
```suggestion
<li>Add the imports to your <code class="inline-code">AppDelegate</code> file under the <code class="inline-code">didFinishLaunchingWithOptions</code> method, as shown:
```

Comment (ameypv-rudder) [reply docs/destinations/attribution/adjust.mdx:120]: We're keeping it consistent with the steps present in the Firebase doc.

**Review by pallabmaiti — APPROVED** [review]

**Pull request merged**